data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -73,10 +66,18 @@ podman run -d --net host --privileged --name httpd \
      --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
      -v $IRONIC_SHARED_VOLUME:/shared:z --entrypoint /bin/runhttpd ${IRONIC_IMAGE}
 
-# Set CACHEURL to the default route, so we try to consume any images cached on the host
-# running the VM (dev-scripts configures a cache here), if none is found then the
-# downloader containers just skip and download from the internet location
-CACHEURL="http://$(ip r | grep $PROVISIONING_NIC | awk '/default/ {print $3};')/images"
+{{ if .PlatformData.BareMetal.ProvisioningIPv6 }}
+IPTABLES=ip6tables
+{{ else }}
+IPTABLES=iptables
+{{ end }}
+
+
+# Set CACHEURL to the the same IP as is used in RHCOS_BOOT_IMAGE_URL, assuming any cache would
+# be the same host, if none is found then the downloader containers just skip and download
+# from the internet location ( IP=n.n.n.n:nn or [x:x::x]:nn )
+IP=$(echo $RHCOS_BOOT_IMAGE_URL | sed -e 's/.*:\/\/\([^/]*\)\/.*/\1/g' )
+CACHEURL="http://$IP/images"
```

Comment:
Since https://github.com/openshift/ironic-rhcos-downloader/pull/24 this is no longer used, but I guess we can remove that in a follow-up PR